InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -53,7 +55,8 @@ def __init__(self,
                  verbose: bool = False,
                  slide_dataset: SlidesDataset = None,
                  tile_size: int = 224,
-                 level: int = 1) -> None:
+                 level: int = 1,
+                 class_names: List[str] = None) -> None:
```

Comment:
Type annotation has a small discrepancy - if you set the default to None, it should be "Optional[List[str]]".

Reply:
Thanks, now changed